mantidimaging/gui/widgets/spectrum_widgets/fitting_display_widget.py

```diff
@@ -13,14 +14,53 @@ class FittingDisplayWidget(QWidget):
     def __init__(self, parent: QWidget | None = None) -> None:
         super().__init__(parent)
 
-        self.layout: QVBoxLayout = QVBoxLayout(self)
-        self.spectrum_plot: SpectrumPlotWidget = SpectrumPlotWidget()
+        self.layout = QVBoxLayout(self)
+        self.spectrum_plot = SpectrumPlotWidget()
         self.layout.addWidget(self.spectrum_plot)
 
+        self.fitting_region = RectROI([0, 0], [1, 1], pen=mkPen((255, 0, 0), width=2), movable=True)
+        self.fitting_region.setZValue(10)
+        self.fitting_region.addScaleHandle([1, 1], [0, 0])
+        self.fitting_region.addScaleHandle([0, 0], [1, 1])
+        self.fitting_region.addScaleHandle([0, 1], [1, 0])
+        self.fitting_region.addScaleHandle([1, 0], [0, 1])
+        self.spectrum_plot.spectrum.addItem(self.fitting_region)
+
     def update_plot(self, x_data: np.ndarray, y_data: np.ndarray, label: str = "ROI") -> None:
+        if x_data is None or x_data.size == 0:
+            return
         self.spectrum_plot.spectrum.clear()
         self.spectrum_plot.spectrum.plot(x_data, y_data, name=label, pen=(255, 255, 0))
+        self.spectrum_plot.spectrum.addItem(self.fitting_region)
+        self.set_default_region(x_data, y_data)
 
     def update_labels(self, wavelength_range: tuple[float, float] | None = None) -> None:
-        if wavelength_range is not None and len(wavelength_range) == 2:
+        """Update wavelength range label below the plot, if available."""
+        if wavelength_range and len(wavelength_range) == 2:
             self.spectrum_plot.set_wavelength_range_label(*wavelength_range)
+
+    def set_default_region(self, x_data: np.ndarray, y_data: np.ndarray) -> None:
+        """Position the ROI centrally over the plotted data."""
+        x_min, x_max = float(np.min(x_data)), float(np.max(x_data))
+        y_min, y_max = float(np.min(y_data)), float(np.max(y_data))
+        x_span = max((x_max - x_min) * 0.25, 20.0)
+        y_span = max((y_max - y_min) * 0.5, 10.0)
+        x_start = (x_max + x_min) / 2 - x_span / 2
+        y_start = (y_max + y_min) / 2 - y_span / 2
+
+        if self.fitting_region is not None:
```

Comment:
`if` should not been needed any more as `self.fitting_region` is never `None`. Same in `set_selected_fit_region`.